src/KubernetesClient/Informers/ListerWatcher.cs

```diff
@@ -0,0 +1,48 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Rest;
+using k8s;
+using System;
+
+namespace k8s.informers
+{    
+    public delegate Task<HttpOperationResponse<L>> ListerD<T, L>();
+    public delegate Watcher<T> WatcherD<T, L>(Action<WatchEventType, T> onEvent, Action<Exception> onError = null, Action onClosed = null);
+
+    public class ListerWatcher<T, L>
+    {           
+        public ListerD<T, L> Lister;
```

Comment:
Why are these public?

Reply:
Reflector code calls the ListWatcher's List and Watcher, hence they are kept as public.